--- a/libclamav/readdb.c	2026-03-06 00:53:41.881820323 +0700
+++ b/libclamav/readdb.c	2026-03-06 00:53:59.814820323 +0700
@@ -126,13 +126,13 @@
         return newname;
     }
 
-    newname = (char *)malloc(namelen + 11 + 1);
+    newname = (char *)malloc(namelen + 9 + 1);
     if (!newname) {
         cli_errmsg("cli_virname: Can't allocate memory for newname\n");
         return NULL;
     }
     memcpy(newname, virname, namelen);
-    memcpy(newname + namelen, ".UNOFFICIAL", 12); /* includes '\0' */
+    memcpy(newname + namelen, ".THAIZONE", 10); /* includes '\0' */
     return newname;
 }
 
@@ -1142,9 +1142,9 @@
             nsz = pt - signame;
         else
             return NULL;
-    } else if (nsz > 11) {
-        if (!strncmp(signame + nsz - 11, ".UNOFFICIAL", 11))
-            nsz -= 11;
+    } else if (nsz > 9) {
+        if (!strncmp(signame + nsz - 9, ".THAIZONE", 9))
+            nsz -= 9;
         else
             return NULL;
     } else if (nsz > 2)
@@ -3878,7 +3878,7 @@
         return CL_EMEM;
     }
 
-    snprintf(newident, strlen(rule->identifier) + 5 + 1, "YARA.%s", rule->identifier);
+    snprintf(newident, strlen(rule->identifier) + 5 + 1, "%s.NSFW", rule->identifier);
 
     if (engine->cb_sigload && engine->cb_sigload("yara", newident, ~options & CL_DB_OFFICIAL, engine->cb_sigload_ctx)) {
         cli_dbgmsg("cli_loadyara: skipping %s due to callback\n", newident);
